content/news/2024-09-30-maplibre-newsletter-september-2024/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "MapLibre Newsletter September 2024"
+date: "2024-09-30"
+categories: ["newsletter"]
+authors: [ramyaragupathy, ianthetechie, harel, bart]
+draft: false
+---
+
+# MapLibre Newsletter - September 2024
+
+## SwiftUI DSL
+
+MapLibre has a new hosted project: the [SwiftUI DSL for MapLibre native](https://github.com/maplibre/swiftui-dsl)! Originally started by Ian Wagner from [Stadia Maps](https://stadiamaps.com/), the goal of the project is to give MapLibre developers on iOS an experience that rivals the latest MapKit for SwiftUI APIs.
+
+[Jacob Fielding](https://github.com/archdoog) ([Rallista](https://rallista.app/)), [Patrick Wolowicz](https://subzero.eu/) and [Patrick Kladek](https://github.com/Patrick-Kladek) ([HudHud](https://hudhud.sa/en)) shared the same vision, and have each contributed significantly to the ongoing development over the last year. This is a great example of the collaboration that MapLibre seeks to enable across the industry, and the authors are excited to have the project officially hosted under the MapLibre organization.
+
+If you'd like to help shape the future of MapLibre in SwiftUI, join us in the `#maplibre-swiftui-compose-playground` channel on [Slack](https://slack.openstreetmap.us/).
+
+## MapLibre GL JS
+
+
+In this month three version were released: 4.7.0, 4.7.1 and 5.0.0-pre.1.
+The prerelease of version 5 is meant to get some feedback on the newly added globe feature!
+We are also planning some breaking changes as part of this version which you can find here, most of them are not disruptive:
+
+- https://github.com/maplibre/maplibre-gl-js/issues/3834
+
+Apart from the exiting news about the globe finally making its way to the main branch there are also some experiments to allow controlling the roll angle of the map to facilitate for some aviation use cases.
+![image](https://github.com/user-attachments/assets/0a0771e6-11d9-40b4-9c49-409733b420dc)
+
+More about it can be found here, there are bounties related to this effort as well if you are interested in developing this, or if your company needs this feature you can chip-in for the costs:
```

Comment:
```suggestion
More about it can be found in the linked issue below, there are bounties related to this effort as well if you are interested in developing this, or if your company needs this feature you can chip-in for the costs:
```